pkg/network/render.go

```diff
@@ -389,3 +393,48 @@ func RenderMultusAdmissionController(conf *operv1.NetworkSpec, manifestDir strin
 	out = append(out, objs...)
 	return out, nil
 }
+
+func ClearNetAttachDef(client cli.Client, prev *operv1.NetworkSpec, conf *operv1.NetworkSpec) error {
```

Comment:
nit: This should be in `pkg/controller/operconfig_controller`, not `pkg/network`.

Reply:
also, can you add a docblock describing this?

Reply:
changed.